src/util/cmdlineargs.cpp

```diff
@@ -97,14 +113,28 @@ warnings and errors to the console unless this is set properly.\n", stdout);
 when a critical error occurs unless this is set properly.\n", stdout);
             }
             i++;
-        } else if (QString::fromLocal8Bit(argv[i]).contains("--midiDebug", Qt::CaseInsensitive) ||
-                   QString::fromLocal8Bit(argv[i]).contains("--controllerDebug", Qt::CaseInsensitive)) {
+        } else if (argv[i] == QString("--useVuMeterGL")) {
+            if (i + 1 == argc) {
+                throw std::runtime_error("Missing argument after --useVuMeterGL");
+            }
+            QString qs(argv[i + 1]);
+            if (qs != "yes" && qs != "no") {
+                throw std::runtime_error("Expected yes or no after --useVuMeterGL");
+            }
+            m_useVuMeterGL = (qs == "yes");
```

Comment:
This would exit (at least in debug builds?), so please refactor to use `fputs` like above.

Reply:
No, these exceptions are caught in main and handled appropriately.

I initially used fputs here and returned false, but with that the entire help text is displayed after the error message, making the error message easy to overlook. But I guess if someone is explicitly adding this option on the command line he will figure that out. I'll make the change.

Reply:
I brought it in line with the error reporting of invalid loglevel arguments. I don't think it's ideal but at least it's consistent.